src/app/wallet-adapter/page.tsx

```diff
@@ -1,10 +1,157 @@
-        <h1>wallet adapter page</h1>
-        <p>this will fetch the info from user&apos;s wallet</p>
-      </div>
+    <div className="w-screen ">
+
+      <TopBar/>
+
+      <section className="w-full py-12 md:py-24 lg:py-32 xl:py-30">
+        <div className="container px-4 md:px-6">
+          <div className="flex flex-col items-center space-y-4 text-center">
+            <div className="space-y-2">
+              <span className="inline-block bg-blue-100 text-gray-600 py-1 px-3 rounded-full text-sm font-semibold mb-4">
+                TipLink Wallet Adapter
+              </span>
+              <h1 className="text-3xl font-bold tracking-tighter sm:text-4xl md:text-5xl lg:text-6xl/none">
+                Making Solana apps{' '}
+                <span className="text-black/70">consumer ready</span>
+              </h1>
+              <p className="mx-auto max-w-[700px] text-gray-500 md:text-xl dark:text-gray-400">
+                Let your users login with just a Google account and start
+                signing transactions.
+              </p>
+            </div>
+            <div className="space-x-4">
+              <Button>
+                Integrate Now
+                <ArrowRight className="ml-2 h-4 w-4" />
+              </Button>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section className="py-20 px-4 bg-gray-200">
+        <div className="max-w-6xl mx-auto ">
+          <h2 className="text-3xl font-bold mb-12 text-center">Key Features</h2>
+          <div className="grid grid-cols-1 md:grid-cols-3 gap-8">
+            {[
+              {
+                icon: Zap,
+                title: 'Easy Integration',
+                description:
+                  'Seamlessly integrate with your existing Solana dApp in minutes.',
+              },
+              {
+                icon: Shield,
+                title: 'Secure Transactions',
+                description:
+                  'Built-in security features to protect user funds and data.',
+              },
+              {
+                icon: Users,
+                title: 'Multi-Wallet Support',
+                description:
+                  'Support for multiple wallet providers, giving users more options.',
+              },
+              {
+                icon: Code,
+                title: 'Developer Friendly',
+                description:
+                  'Comprehensive documentation and examples to get you started quickly.',
+              },
+              {
+                icon: DollarSign,
+                title: 'Low Fees',
+                description:
+                  'Minimize transaction costs for your users with optimized fee structures.',
+              },
+              {
+                icon: Star,
+                title: 'Customizable UI',
+                description:
+                  'Fully customizable UI to match your brand and user experience.',
```

Comment:
We can really keep the code more presentable by keeping this array of objects above the return statement, makes code more readable

Reply:
ok.